spring-restdocs-core/src/main/java/org/springframework/restdocs/RestDocumentation.java

```diff
@@ -34,12 +34,10 @@
 
 	/**
 	 * Creates a new {@code RestDocumentation} instance that will generate snippets to the
-	 * given {@code outputDirectory}.
-	 *
-	 * @param outputDirectory the output directory
+	 * to &lt;gradle/maven build path&gt;/generated-snippet if no runtime property ({@code snippetOutputDirectory}) is set.
 	 */
-	public RestDocumentation(String outputDirectory) {
```

Comment:
While this class is deprecated (and might be removed before 1.2 is released), for the time being at least this constructor needs to be kept, both to allow the output directory to be overridden and to maintain backwards compatibility.